toolkit/scalebar/src/main/java/com/arcgismaps/toolkit/scalebar/internal/ScalebarRenderer.kt

```diff
@@ -137,6 +216,32 @@ internal fun LineScaleBarPreview() {
     }
 }
 
+@Preview(showBackground = true, backgroundColor = 0xff91d2ff)
+@Composable
+internal fun GraduatedScaleBarPreview() {
+    val maxWidth = 500f
+    val tickMarks = listOf(
+        ScalebarDivision(0, 0.0, 0.0, "0"),
+        ScalebarDivision(1, (maxWidth / 4.0), 0.0, "25"),
+        ScalebarDivision(2, maxWidth / 2.0, 0.0, "50"),
+        ScalebarDivision(3, (maxWidth / 4.0)* 3, 0.0, "75"),
+        ScalebarDivision(4, maxWidth.toDouble(), 0.0, "100")
```

Comment:
```suggestion
        ScalebarDivision(4, maxWidth.toDouble(), 0.0, "100 km")
```
the rendering logic should work with the last value having the units included in the text